File: day-7/StarPattern.java
import java.util.Scanner;

public class StarPattern {
    public static void main(String[] args) {
        System.out.println("Enter your limit :");
        try (Scanner s = new Scanner(System.in)) {
            int limit=s.nextInt();

            for(int i=1;i<limit;i++){

                for(int j=0;j<i;j++){
                    System.out.print("* ");
                }
                System.out.print("\n");
            }
        }

    }
    
}

File: day-4/star-pattern.c
#include <stdio.h>
#include <stdlib.h>

int main(void){
    int limit,j,i;
    printf("Enter your limit");
    scanf("%d",&limit);
    for (i=1; i<=limit; i++)
    {
        for (j=0;j<i;j++)
        {
           printf("* ");
        }
        printf("\n");
        
    }
    









    return EXIT_SUCCESS;

}